src/classes/GE_DefaultTemplate.cls

```diff
@@ -0,0 +1,190 @@
+        List<FORM_Section> sectionList = defaultLayout.sections;
+
+        for (FORM_Section section : sectionList) {
+            List<FORM_Element> elementList = section.elements;
+
+            for (FORM_Element element : elementList) {
+                if (element.elementType == ELEMENT_TYPE_FIELD) {
+
+                    // Get the updated dev name for this org and replace it into the Base JSON
+                    String partialDevName = element.dataImportFieldMappingDevNames.get(0).substringBeforeLast('_');
+
+                    // If the updated Dev Name map doesn't contain the partial dev name
+                    // it is not necessary to be updated. It is probably a field
+                    // that is on the DI itself and not a mapped field.
+                    // e.g. 'Donation_Donor__c', 'Account1Imported__c', etc
+                    if (updatedDevNameMap.containsKey(partialDevName)) {
+                        element.dataImportFieldMappingDevNames.set(0, updatedDevNameMap.get(partialDevName));
+                    } else {
+                        String fullDevName = element.dataImportFieldMappingDevNames.get(0);
+
+                        if (fullDevName.endsWith('__c')) {
+                            fullDevName = UTIL_Namespace.StrTokenNSPrefix(fullDevName);
+                        }
+
+                        // If it is a DI field, apply the namespace, if there is one. 
+                        element.dataImportFieldMappingDevNames.set(0, fullDevName);
+                    }
+                }
+            }
+        }
+
+        // Update the batch header fields with the namespace prefix, if necessary
+        List<FORM_BatchHeaderField> batchHeaderFieldList = defaultTemplate.batchHeaderFields;
+        
+        for (FORM_BatchHeaderField batchHeaderField : batchHeaderFieldList) {
+
+            if (batchHeaderField.apiName.endsWith('__c')) {
+                batchHeaderField.apiName = UTIL_Namespace.StrTokenNSPrefix(batchHeaderField.apiName);
+            }
+        }
+
+        newDefaultTemplate.Name = defaultTemplate.name;
+        newDefaultTemplate.Description__c = defaultTemplate.description;
+        newDefaultTemplate.Template_JSON__c = JSON.serialize(defaultTemplate);
+        newDefaultTemplate.Format_Version__c = '1.0';
+        return newDefaultTemplate;
+    }
+
+    /**
+     * @description This method builds a mapping of partial dev name to the full dev name using the org's field mapping custom metadata records. 
+     *
+     * @return A mapping of partial dev name to full dev name. e.g. Account1_Name -> Account1_Name_ee63cf9df
+     */
+    public static Map<String, String> getCurrentDevNameMapping() {
+        // Get the org's migrated field mappings. 
+        List<Data_Import_Field_Mapping__mdt> fieldMappingCMTList = [Select Id, MasterLabel, DeveloperName, Target_Field_API_Name__c, Source_Field_API_Name__c, Target_Object_Mapping__r.Object_API_Name__c
```

Comment:
Can you adjust the format of this SOQL query (capitalize SELECT and WHERE and break each field onto its own line)? There should be good examples to draw from elsewhere